### DISCUSSION TITLE
Get some statistic date from moose in phaseFiled/grain growth model

### DISCUSSION THREAD

**Discussion by zengfy-hust** [idaholab/moose#18066] [Q&A General] [answered]
Hello Moose experts:
         I recently get the data i want from moose .however ,except for the visulize file (.e)file .I can also get avg_grain_volume  etc,however ,Can I get the data of each grain's diameter  evlove with time  ？ Thank you .

**Comment by GiudGiud** [comment, marked as answer]
Hello

What kind of object is this avg_grain_volume? If it s a post processor, then it s stored as a global variable in the exodus file and you can view it in paraview

Guillaume
